server/models/Document.ts

```diff
@@ -527,8 +527,8 @@ class Document extends ParanoidModel {
     "teamId" = :teamId AND
     "collectionId" IN(:collectionIds) AND
     ${documentClause}
-    "deletedAt" IS NULL AND
-    "publishedAt" IS NOT NULL
+    ${!options.includeDrafts ? '"publishedAt" IS NOT NULL AND' : ""}
+    "deletedAt" IS NULL
```

Comment:
So I think this is a mistake, we're within the `searchForTeam`  method here which has no user context, so there are no drafts we can ever show.